x/examples/fetch-speed/main.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/Jigsaw-Code/outline-sdk/x/config"
+)
+
+var debugLog log.Logger = *log.New(io.Discard, "", 0)
+
+func init() {
+	flag.Usage = func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage: %s [flags...] <url>\n", path.Base(os.Args[0]))
+		flag.PrintDefaults()
+	}
+}
+
+func main() {
+	verboseFlag := flag.Bool("v", false, "Enable debug output")
+	transportFlag := flag.String("transport", "", "Transport config")
+	addressFlag := flag.String("address", "", "Address to connect to. If empty, use the URL authority")
```

Comment:
FYI, with the "override" config, you no longer need this flag and the special logic. You could just override it in the transport flag. See https://pkg.go.dev/github.com/Jigsaw-Code/outline-sdk/x/config#hdr-Examples

Reply:
I addressed the requested changes except this one. 

I will do some tests and submit another MR soon. I will add the timeout value as a Flag for example.

Reply:
Sounds good, we can leave that to another change.